src/styles/index.scss

```diff
@@ -1,3 +1,50 @@
 body {
   margin: 0;
 }
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: calc($cell-size * 7 + $gap * 6);
+  gap: $gap;
+  padding: 10px;
+
+  &__day {
+    width: $cell-size;
+    height: $cell-size;
+    border: 1px black solid;
+    box-sizing: border-box;
+    background-color: #eee;
+    transition: 0.5s;
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 30px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    line-height: 30px;
+  }
+
+  @for $day from 1 through 31 {
+    &__day:nth-child(#{$day})::before {
+      content: '#{$day}';
+    }
+  }
+
+  @for $length from 28 through 31 {
+    &--month-length-#{$length} > :nth-child(n + #{$length + 1}) {
+      display: none;
+    }
+  }
+
+  @each $name, $day in $days {
```

Comment:
The variable `$days` is used here, but it's not defined within this file. Ensure that `$days` is defined before it's used, or import the file where it's defined.